### PR TITLE
feat(shop): add prdoucts table display and product card

### PR DESCRIPTION
**Pull request by m-spilman**

### CODE REVIEW

File: src/components/add-products/add-products.js [hunk 1]
```diff
@@ -74,13 +75,19 @@ function AddProducts() {
...
       <div>
         <h2>Add New Product</h2>
-        <form onSubmit={onSubmit}>
-          <BuildForm fields={fields} />
-          <div className="form-group">
-            <button className="btn btn-primary">Add</button>
+        <form onSubmit={onSubmit} className="row">
+          <div className="col-8">
+            <BuildForm fields={fields} />
+            <br />
```
Comment (ahaly92) [inline src/components/add-products/add-products.js]: generally you wanna use css over` <br/>`

File: src/app.scss [hunk 2]
```diff
@@ -1,3 +1,3 @@
 .App {
```
Comment (BarretteBanner) [inline src/app.scss]: let's try to use lowercase. Conventions vary but generally title case is reserved for global css

File: src/components/add-products/product-card.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+function ProductCard(props) {
+  const style = { width: '18rem' };
```
Comment (ahaly92) [inline src/components/add-products/product-card.js]: let's stay away from inline styles. move css to scss files and use classes

File: src/components/form-builder.js [hunk 4]
```diff
@@ -24,14 +24,23 @@ function BuildForm(props) {
...
-              onChange={field.onChange}
-              className={field.className}
-            />
+            <div className="input-group">
+              {field.name === 'price' ? (
+                <div className="input-group-prepend">
+                  <span className="input-group-text">$</span>
+                </div>
+              ) : (
+                <div></div>
```
Comment (ahaly92) [inline src/components/form-builder.js]: you can return null here

**Review by BarretteBanner — APPROVED** [review]

File: src/components/form-builder.js [hunk 5]
```diff
@@ -24,14 +24,23 @@ function BuildForm(props) {
...
-              onChange={field.onChange}
-              className={field.className}
-            />
+            <div className="input-group">
+              {field.name === 'price' ? (
+                <div className="input-group-prepend">
+                  <span className="input-group-text">$</span>
+                </div>
+              ) : (
+                <div>{null}</div>
```
Comment (ahaly92) [inline src/components/form-builder.js]: sorry i meant remove this div completely and replace it with null since there's nothing inside of it

**Review by ahaly92 — APPROVED** [review]
one small thing and then good to go

**Review by ahaly92 — APPROVED** [review]

**Pull request merged**